mayastor/src/bin/cli/pool_cli.rs

```diff
@@ -0,0 +1,163 @@
+use super::context::Context;
+use ::rpc::mayastor as rpc;
+use byte_unit::Byte;
+use clap::{App, AppSettings, Arg, ArgMatches, SubCommand};
+use tonic::{Code, Status};
+
+pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
+    let create = SubCommand::with_name("create")
+        .about("Create storage pool")
+        .arg(
+            Arg::with_name("block-size")
+                .short("b")
+                .long("block-size")
+                .value_name("NUMBER")
```

Comment:
Should value name be size?

Reply:
the size is a number...  I see your thinking, but this was as it was before and I've not touched the usage messages etc..

Reply:
Ok :+1: